<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToolDialog</class>
 <widget class="QDialog" name="ImageToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert Image</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="imageLinkLabel">
     <property name="text">
      <string>Image Link:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="imageLinkEdit">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>http://example.com/image.jpg</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseFileButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="alternateTextLabel">
     <property name="text">
      <string>Alternate Text:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="alternateTextEdit">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="optionalTitleLabel">
     <property name="text">
      <string>Optional Title:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="optionalTitleEdit">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>imageLinkEdit</tabstop>
  <tabstop>chooseFileButton</tabstop>
  <tabstop>alternateTextEdit</tabstop>
  <tabstop>optionalTitleEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImageToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImageToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseFileButton</sender>
   <signal>clicked()</signal>
   <receiver>ImageToolDialog</receiver>
   <slot>chooseFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chooseFileButtonClicked()</slot>
 </slots>
</ui>
